lib/rust/ensogl/example/complex-shape-system/src/lib.rs

```diff
@@ -155,9 +155,9 @@ pub fn main() {
                 // DEBUG!("{scene.layers.node_searcher_mask:#?}");
                 // DEBUG!("{scene.layers.viz:#?}");
```

Comment:
Commented code.

Reply:
These are left intentionally, there is a comment for it: `// These comments are left for easy debugging in the future.`. Its not a new code and I think these should be left there.